packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -397,16 +398,20 @@ function addEditableCallbacks<T extends DbObject>({collection, options = {}}: {
       // after-create callback, passing through an intermediate state where it's
       // undefined; and it's missing _id and schemaVersion, which leads to having
       // ObjectID types in the database which can cause problems.
-      const firstRevision = await Connectors.create(Revisions, newRevision as DbRevision);
-      
+      const firstRevision = await createMutator({
+        collection: Revisions,
+        document: newRevision,
+        validate: false
+      });
```

Comment:
This seems to work :) I initially included validation but that caused this to fail. I couldn't find any validation specific to Revisions, just saw the generic auth ones, and it wasn't clear to me if this was a case where we **should** be ignoring that validation since it's a system-generated document. So I left it like this since it works, feel free to change it.